ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -27,10 +28,28 @@ SplitView {
         chatColumn.onActivated()
     }
 
-    Loader {
-        id: contactColumnLoader
-        SplitView.preferredWidth: Style.current.leftTabPreferredSize
-        sourceComponent: appSettings.communitiesEnabled && chatsModel.communities.activeCommunity.active ? communtiyColumnComponent : contactsColumnComponent
+    leftPanel: Item {
+        anchors.fill: parent
+
+        Loader {
+            id: contactColumnLoader
+            anchors.top: parent.top
+            anchors.bottom: parent.bottom
+            width: parent.width
+            anchors.horizontalCenter: parent.horizontalCenter
+            sourceComponent: appSettings.communitiesEnabled && chatsModel.communities.activeCommunity.active ? communtiyColumnComponent : contactsColumnComponent
+        }
+    }
+
+    rightPanel: Item {
+        anchors.fill: parent
```

Comment:
The Item doesn't seem useful. You could just put the ChatColumn I think

Reply:
Thanks, good catch!